### DISCUSSION TITLE
Artist signatures on art contest images are cut off on corners, worse in lazer

### DISCUSSION THREAD

**Discussion by CloudyCrayons** [ppy/osu#18827] [Ideas] [answered]
let me start by saying I'm fairly confident most of the community wants artists to be able to put signatures on their summited artwork, or at least have a method to give credit, but in the current state it does look awkward for half the signature to be cut off in the corners of the artwork.

This is currently worse in lazer because the background shifts around when options and notifications are opened (even when parallax is disabled) and the art must be zoomed in to allow for that.

Best solution is just to have a small Info button on the main menu for artists to submit their pixivs/twitters or even their osu! profiles . this also prevents artists from making too flashy signatures.

But I'm not currently aware of the developers thoughts on signatures in the first place. For some reason I recall the submission page mentioning keeping signatures subtle but I don't see that now.

examples

![image](https://user-images.githubusercontent.com/32446731/175458859-96821da0-f9ab-41c6-b7ab-f28299087620.png)

![image](https://user-images.githubusercontent.com/32446731/175458881-6b56d5f4-cb6c-4184-aafb-137326643320.png)

Thank you for reading

**Comment by peppy** [comment, marked as answer]
This is an issue with the artist, not us. They could give their signature more padding.
